test/renaissance_web/controllers/auction_controller_test.exs

```diff
@@ -20,81 +36,65 @@ defmodule RenaissanceWeb.AuctionControllerTest do
     assert html_response(conn, 200) =~ "Create an Auction"
   end
 
-  test "POST /auctions/new with valid params creates an auction" do
+  test "POST /auctions with valid params creates an auction" do
     conn =
       login()
-      |> post("/auctions/new", %{
-        "auction" => %{
-          title: "Test Title",
-          description: "Test description.",
-          end_date_day: "3019-04-15",
-          end_date_time: "14:03",
-          price: "10.00"
-        }
-      })
+      |> post("/auctions", @auction_one)
 
     assert get_flash(conn, :info) == "Auction Created!"
   end
 
-  test "POST /auctions/new fails to create auction when title is blank" do
+  test "POST /auctions fails to create auction when title is blank" do
```

Comment:
Couldn't you do `%{@aucton_one | title: ""} or however to merge the maps?

Reply:
Resolved with 159a775e493da7136d042a4f76b690b4d8cb2aea